fvm/evm/stdlib/contract_test.go

```diff
@@ -236,6 +236,237 @@ func newEVMScriptEnvironment(handler types.ContractHandler, service flow.Address
 	return scriptEnvironment
 }
 
+func TestEVMEncodeABI(t *testing.T) {
+
+	t.Parallel()
+
+	handler := &testContractHandler{}
+
+	contractsAddress := flow.BytesToAddress([]byte{0x1})
+
+	transactionEnvironment := newEVMTransactionEnvironment(handler, contractsAddress)
+	scriptEnvironment := newEVMScriptEnvironment(handler, contractsAddress)
+
+	rt := runtime.NewInterpreterRuntime(runtime.Config{})
+
+	script := []byte(`
+      import EVM from 0x1
+
+      access(all)
+      fun main(): [UInt8] {
+        // bytes for address 0x7A58c0Be72BE218B41C608b7Fe7C5bB630736C71
+        let address = EVM.EVMAddress(
+          bytes: [122, 88, 192, 190, 114, 190, 33, 139, 65, 198, 8, 183, 254, 124, 91, 182, 48, 115, 108, 113]
+        )
+        var data = EVM.encodeABI([address])
+        var values = EVM.decodeABI([Type<EVM.EVMAddress>()], data: data)
+        assert(values.length == 1)
+        assert((values[0] as! [UInt8; 20]) == address.bytes)
```

Comment:
The `EVM.decodeABI` for `Type<EVM.EVMAddress>()` should actually return a `EVM.EVMAddress` struct, and not just the `bytes`.